src/Search_Replace_Command.php

```diff
@@ -108,7 +116,50 @@ class Search_Replace_Command extends WP_CLI_Command {
 	 * : Produce report. Defaults to true.
 	 *
 	 * [--report-changed-only]
-	 * : Report changed fields only. Defaults to false.
+	 * : Report changed fields only. Defaults to false, unless logging, when it defaults to true.
+	 *
+	 * [--log[=<file>]]
+	 * : Log the items changed. If <file> is not supplied or is "-", will output to STDOUT.
+	 * Warning: causes a significant slow down, similar or worse to enabling --precise or --regex.
+	 *
+	 * [--log_context=<before_num,after_num>]
+	 * : Number of characters to display before and after the match. One number sets both before and after. Two comma-separated numbers set before and after respectively. Defaults to 40. Ignored if not logging.
```

Comment:
1. For consistency, should we follow `--before_context=<num>` and `--after_context=<num>` as established by `wp db search` ? I rather like `--log_context=<>` better, but I think we should opt for consistency whenever possible.
2. We should wrap the argument description at whatever count we use for the others.

Reply:
> we should opt for consistency

Am going to propose adding the same "shortcut" version `--context` to `wp db search` (also a shortcut `---colors`). This would make them more or less consistent (barring the `log_` prefix), with `db search` having a superset.

(The main motivation for not just using `log_context_before` etc was as you mention below the large amount of options that `search-replace` already has.)

The `log_` prefix seemed needed to give clarity to and group the options and is sort of consistent with `db search` (which doesn't have a `log` option) but I'll drop if you think it's confusing/inconsistent.

> wrap the argument description

Will do.

(Probably the same/similar newline stripping as is now done for the handbook should be done for help in general, as the some-do, some-don't wrapping currently on help looks odd.)